envoy/cilium/npds.proto

```diff
@@ -47,6 +47,10 @@ message NetworkPolicy {
   // combination.
   // Optional. If empty, all flows in this direction are denied.
   repeated PortNetworkPolicy egress_per_port_policies = 3;
+
+  // The addresses which the network policy applies to.
+  // Required.
+  repeated string addresses = 4;
```

Comment:
The idea was to replace (or complement) the `uint64 policy = 1` field above with `string host_address = 1`, which would contain the IP address of the endpoint.